python/lsst/daf/butler/remote_butler/server_models.py

```diff
@@ -416,3 +417,17 @@ class QueryAllDatasetsRequestModel(pydantic.BaseModel):
     bind: dict[str, ColumnLiteral]
     limit: int | None
     with_dimension_records: bool
+
+
+class GetFileTransferInfoRequestModel(pydantic.BaseModel):
+    MAX_ITEMS_PER_REQUEST: ClassVar[int] = 10000
```

Comment:
```suggestion
    MAX_ITEMS_PER_REQUEST: ClassVar[int] = 10_000
```
Slightly more readable at a glance if we use the underscore separators.